tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3404,7 +3404,7 @@ ruleTester.run('no-unused-prop-types', rule, {
         ': Props): JSX.Element {',
         'const handleVerifySubmit = ({',
         '  otp,',
-        ' }) => {',
+        ' }: {otp:string}) => {',
```

Comment:
why does this need to change?

Reply:
I am embarrassed to say that this is a typo. 😭 
This test case is trying to test the issue from https://github.com/yannickcr/eslint-plugin-react/issues/2689.
I added it back in this pr.
I will change the pr title.